targets/corefx.props

```diff
@@ -14,11 +14,13 @@
   <ItemGroup>
     <RepositoryReference Include="coreclr" />
     <RepositoryReference Include="standard" />
+    <EnvironmentVariables Include="DotNetSourceBuildRestoreSources=$(SourceBuiltPackagesPath)" Condition="'$(OfflineBuild)' != 'true'" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(OfflineBuild)' == 'true'">
```

Comment:
I think we want the `LatestCommit` `OfficialBuildId` `VersionSeedDate` parameters even if we aren't building offline.  Why the change?

Reply:
They are only interesting when building from the tarball. For the LatestCommit we will get the live one from git and for OfficialBuildId/VersionSeedDate we only want that if we are producing an official build and we need it to exactly match one we shipped which again is only interesting for the tar-ball.